ci-operator/step-registry/aws/c2s/instance-metadata/aws-c2s-instance-metadata-ref.yaml

```diff
@@ -0,0 +1,21 @@
+ref:
+  as: aws-c2s-instance-metadata
+  from: upi-installer
+  cli: latest
+  grace_period: 10m
+  commands: aws-c2s-instance-metadata-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  credentials:
+  - namespace: test-credentials
+    name: openshift-custom-mirror-registry
+    mount_path: /var/run/vault/mirror-registry
+  documentation: |-
+    Mirror particular images for C2S and SC2S:
+    * C2S apply metadata patch
+      * https://github.com/staebler/cap-token-refresh/
```

Comment:
Ah I see, this is a github repo. You should fork this. staebler is no longer with rh

Reply:
Updated.